MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -53,6 +56,8 @@ public override SpriteFontContent Process(FontDescription input, ContentProcesso
                         if (File.Exists(fontFile))
                             break;
                     }
+                    if (File.Exists(fontFile))
```

Comment:
There is code for aborting above it, which will abort before you even get to your if...